meetings/2018-11-26.md

```diff
@@ -0,0 +1,98 @@
+  - Sarah noted that the language could be more inclusive as the goal should be one organization working together
+  - Michael noted the Node.js Board seat was omitted and should be the starting point for discussion. Michael noted he wanted to see this and knew others had a similar expectation. Myles noted the TSC should consider who that rep might be and get them ready.
+  - Myles also noted that following the Staging process, this could probably move in and the changes made until Stage 2.
+  - Agreement on the call was to move it to Stage 0 so long as all the parts not currently with consensus will be documented.
+  - This is what I suggest -> Agreement was to take it back to github to discuss board seats
+     and after that is resolved it can land.
+
+* Drafting initial governance for merged foundation [#33](https://github.com/nodejs/bootstrap/issues/33)
+  - 25 minute timebox
+  - Refs:
+    - What does C3 look like? [#38](https://github.com/nodejs/bootstrap/issues/38)
+    - What does the CPC look like? [#37](https://github.com/nodejs/bootstrap/issues/37)
+    - [WIP] PROJECT PROGRESSION [#45](https://github.com/nodejs/bootstrap/pull/45)
+    - doc: initial version of CPC Charter [#48](https://github.com/nodejs/bootstrap/pull/48)
+  - Myles provided an overview of the issues opened to help fill out the presentation with details of the governance
+  - Michael provided an overview of open questions in each issue
+  - Joe walked through the potential areas of focus for the C3. Doc that Adam and others had helped with: https://docs.google.com/document/d/1JvU3RW8zhSw7VEIf5eDO_po4LJL-T2bNjgKrdutfi2s/edit 
+  - including community cultivation, advocacy and support, mentorship, end user feedback and support, inclusivity and diversity, language documentation and auditing, event tooling and conference/meetup resources, social media resources (e.g. guidance on tone, crisis response), swag
+  - Michael noted it was interesting to look at a similar document drafted for the CPC
+   * ensuring collaboration is the driving principle within a Project, between
+     _$FOUNDATION_ Projects, and between _$FOUNDATION_ Projects and the broader
+     community.
+   * defining and maintaining neutral consensus for the technical vision for the _$FOUNDATION_
+   * approving new projects within the scope for _$FOUNDATION_ set by the Governing Board,
+     as outlined in [XXXX], and creating conceptual architecture for how the projects
+     fit together.
+   * aligning projects, removing or archiving projects, as outlined in [XXXX].
+   * defining common practices to be implemented across _$FOUNDATION_ projects, if any.
+   * The CPC is also responsible for mediating technical conflicts between
+     _$FOUNDATION_ Projects when attempts to resolve those conflicts within the Project
+    and were unsuccessful. 
+   * The CPC serves as the _$FOUNDATION_'s primary technical
+     liaison body with external open source pro 
```

Comment:
We lost a bit here.  

 * The CPC serves as the _$FOUNDATION_'s primary technical
     liaison body with external open source projects, consortiums and groups and is
     also responsible for technical participation in standards bodies